drizzlepac/haputils/hapcut_utils.py

```diff
@@ -0,0 +1,383 @@
+            else:
+                shutil.move(file, local_file)
+            # Record what files were downloaded and their current location
+            local_files.append(os.path.basename(local_file))
+        else:
+            local_files.append(file)
+    if not archive:
+        # Remove astroquery created sub-directories
+        shutil.rmtree('mastDownload')
+
+    return(local_files)
+
+
+def make_the_cut(input_files, sky_coord, cutout_size, single_outfile=True, output_dir=".", verbose=False):
+    """
+    This function makes the actual cut in the input MVM drizzled filter-level images. As such it is
+    a high-level interface for the `˜astrocut.cutout.fits_cut` functionality.
+
+    Parameters
+    ----------
+    input_files : list
+        List of fits image filenames from which to create cutouts. The SCI image is assumed to be
+        in the first extension with the weight image in the second extension.
+
+    sky_coord : str or `~astropy.coordinates.SkyCoord` object
+        The position around which to cutout. It may be specified as a string ("ra dec" in degrees)
+        or as the appropriate `~astropy.coordinates.SkyCoord` object.
+
+    cutout_size : int, array-like, `~astropy.units.Quantity`
+        The size of the cutout array. If ``cutout_size`` is a scalar number or a scalar
+        `~astropy.units.Quantity`, then a square cutout of ``cutout_size`` will be created.
+        If ``cutout_size`` has two elements, they should be in ``(ny, nx)`` order.  Scalar numbers
+        in ``cutout_size`` are assumed to be in units of arcseconds. `~astropy.units.Quantity` objects
+        must be in angular units.
+
+    single_outfile : bool
+        Default True. If True, return all cutouts in a single fits file with one cutout per extension,
+        if False return cutouts in individual fits files.  Since both the SCI and WHT extensions of the
+        input files are cutout, individual fits files will contain two image extensions, a SCI followed
+        by the WHT.  Correspondingly, the single fits file will contain a SCI and then a WHT extension
+        for each input image.  If returing a single file the filename will have the form: 
+        <cutout_prefix>_<ra>_<dec>_<size x>_<size y>.fits. If returning multiple files
+        each will be named: <original filemame base>_<ra>_<dec>_<size x>_<size y>.fits.
+        Output filename of the form: hst_skycell_cutout-p<pppp>-ra<##>d<####>-dec<n|s><##>d<####>_detector_filter.fits
+
+    output_dir : str
+        Default value '.'. The directory to save the cutout file(s) to.
+
+    verbose : bool
+        Default False. If True, additional intermediate information is printed.
+
+    Returns
+    -------
+    response : str or list
+        If single_outfile is True returns the single output filepath. Otherwise returns a list of all 
+        the output filepaths.
+        If memory_only is True a list of `~astropy.io.fit.HDUList` objects is returned instead of
```

Comment:
Where is the 'memory_only' parameter set?

Reply:
This parameter is not exposed in the public interface, and this value is hard-coded in make_the_cut() to True.  By setting the value to True when invoking the low-level software, HDUList objects are returned to the high-level software.  This allows the high-level software to modify the FITS file headers before the data is written out.  We had decided we always wanted to write out a file.

Reply:
This can confuse an end-user if they are being told about a parameter which they do not have access to modify.  The phrase 'If memory_only is True' could/should be removed to avoid this confusion.